.github/workflows/main.yml

```diff
@@ -2,81 +2,63 @@ name: Build and publish
 
 on:
   push:
-    branches: [ master ]
+    branches: master
   pull_request:
-    branches: [ master ]
+    branches: master
 
 jobs:
   build_extensions:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Is there a reason for not using the latest version? This is my only question. Everything seems to be much cleaner but I admittedly know nothing about this.